po/common/fr.po

```diff
@@ -4324,7 +4324,7 @@ msgstr "Code-barres invalide"
 # Either 'ltr' for left to right languages like English or 'rtl' for right to left languages like Arabic
 msgctxt "text_direction"
 msgid "ltr"
-msgstr ""
+msgstr "de gauche à droite"
```

Comment:
```suggestion
msgstr ""
```

Reply:
arg. we need to clean it in crowdin, otherwise it will happen again

Reply:
https://crowdin.com/translate/openfoodfacts/1378/en-fr?filter=basic&value=0#q=ltr
fixed

Reply:
at least the test stopped it breaking the site this time :)